Automattic/themes
# Remove legacy Learning Mode CSS as it's part of Sensei now

Fixes #7429.

#### Changes proposed in this Pull Request:
Removes the `learning-mode.css` file as it's part of Sensei now and users have had plenty of time to update both the plugin and the theme. We also want to try to minimize the use of Sensei-specific code in the theme, and instead have it in Sensei where possible.

#### Testing Instructions
1. Checkout https://github.com/Automattic/sensei/pull/7219.
2. View the lessons and quizzes in Learning Mode. There should be no change to the UI.
